### DISCUSSION TITLE
Image Server?

### DISCUSSION THREAD

**Discussion by Tharic99** [meisnate12/Plex-Meta-Manager#36] [Q&A] [answered]
I may be missing something in the wiki docs, but is the Image Server functionality from PAC missing?

https://github.com/mza921/Plex-Auto-Collections#image-server

**Comment by meisnate12** [comment]
It has an asset directory that functions basically the same. I apparently completely forgot to include it in the wiki. There's a way to specify its location I'll have to check the code later to see exactly where but by default it will look inside the config folder for an assets folder. Then it will look for a folder that matches the collection name and looks for a poster.png|jpg and background.png|jog

**Comment by Tharic99** [comment]
Ah okay. I'll poke through the code if I have time today and see if I can find anything. 

Right now, all of my posters are in a /config/posters folder and the name of the file matches the name of the collection. 493 custom posters in there right now. At least I'm on a linux back end, so if I need to move them all into folders and then rename, it's a bit easier.

**Comment by meisnate12** [comment]
I actually wrote a python script to put all the posters in folders named what the poster was named and change the poster name to poster. I'll post it in a bit

**Comment by meisnate12** [comment, marked as answer]
i found it i have it as an attribute under the plex attribute https://github.com/meisnate12/Plex-Meta-Manager/wiki/Plex-Attributes

**Comment by b101marc** [comment]
I noticed when running parts of the template there are paths created for some of the collections. Where did you source those? Would like to add them to my own library.

EX:
```
| Details: have been updated                                                                         |
| poster not found at: /opt/appdata/Plex-Meta-Manager/config/assets/Trending/poster.*                |
| background not found at: /opt/appdata/Plex-Meta-Manager/config/assets/Trending/background.* 
```

**Comment by meisnate12** [comment]
these are the posters I've created for my collections that didn't really have a place on theposterdb.com

https://drive.google.com/drive/folders/1cDv1WGyjCXM4r_Y5orA7YVNHnyT8XoJQ?usp=sharing

**Reply by bobokun** [reply]
Thank you for sharing this! Do you mind sharing the PSD file or original file you used to create the Best of Year posters? This way we can create ones for 2022 onward using the same style?

**Reply by meisnate12** [reply]
the PSD isn't mine to share but you can find it [here](https://www.reddit.com/r/PlexPosters/comments/bqb1bj/collection_tv_networks/)!

**Reply by bobokun** [reply]
Thank you for the links! Can I please ask what font was to generate the best of year?

**Reply by meisnate12** [reply]
I'm not sure off the top of my head but it was one already being used In The PSD

**Comment by Tharic99** [comment]
> i found it i have it as an attribute under the plex attribute https://github.com/meisnate12/Plex-Meta-Manager/wiki/Plex-Attributes

Got it. So the file goes into it's own folder that matches the collection name under the assets. What should the filename be?
Collection name of "Sean Connery" for example. Is it like `/assets/Sean Connery/poster.jpg` ?



> I actually wrote a python script to put all the posters in folders named what the poster was named and change the poster name to poster. I'll post it in a bit

Did you have a chance to post this anywhere too?

**Comment by meisnate12** [comment]
> Got it. So the file goes into it's own folder that matches the collection name under the assets. What should the filename be?
Collection name of "Sean Connery" for example. Is it like /assets/Sean Connery/poster.jpg ?

Yes it looks for poster.png, poster.jpg, or poster.jpeg inside the collection folder.

>Did you have a chance to post this anywhere too?

I actually deleted it so I had to rewrite it here.

https://github.com/meisnate12/Plex-Meta-Manager-Configs/discussions/8
